### ISSUE TITLE
minimize app

### ISSUE THREAD

**Issue by trisosamu** [clesssalvein/GlpiClient#2]
When I open exe I don't see nothing. I must go to system tray and relaunch app

**Comment by clesssalvein** [comment]
Yes, that was the plan. The application is minimized to the tray at startup. Can you open the app by double-clicking on the tray icon?

**Comment by trisosamu** [comment]
Yes, but for user it can be useful open app and create immediately ticket without going into tray icon. is it possibile to create a windows to choose one of the two option?

**Comment by clesssalvein** [comment]
Do you want the app window to open when the OS starts?

**Comment by trisosamu** [comment]
in my case, I put the icon shortcut on the desktop, and I would like that when I click on it, the credential request window opening immediately. without having to open tray icon


![image](https://user-images.githubusercontent.com/78987040/213699858-40b434f4-7c0f-4f77-83b3-1f64ea512db8.png)

**Comment by clesssalvein** [comment]
Ok. I'll add an option to be able not to minimize the app window to tray after app startup.

**Comment by trisosamu** [comment]
thanks

**Comment by clesssalvein** [comment]
Option to choose whether or not to minimize to tray the app window at startup added in release [GlpiClient v0.304](https://github.com/clesssalvein/GlpiClient/releases/tag/v0.304)

**Comment by trisosamu** [comment]
It works

**Comment by trisosamu** [comment]
Last thing. Is it possible add settings button called "not minimize app to tray ever" ? To bring app always on application bar?

**Comment by trisosamu** [comment]
I'm trying to edit GlpiClient.py to resolve my problems himself, but when I run script, there are these error,

![image](https://user-images.githubusercontent.com/78987040/214047211-17be190e-051b-40d4-843b-a36be17eb3be.png)

**Comment by clesssalvein** [comment]
Please write how you think the program should behave when you press the "minimize" button and the "close" button. I'll try to add additional options to the app.

**Comment by trisosamu** [comment]
When I press minimize button --> minimize in the application bar  (as now)
![image](https://user-images.githubusercontent.com/78987040/214231807-81d3384b-d7e1-459d-9f2c-48a6d88a1ad2.png)

When i press close button -->  minimize into tray  (as now)

But If I press on the .exe (second time) I would like that application open from tray, without showing popup "application already started" (like microsoft teams )

**Comment by clesssalvein** [comment]
Option to choose whether to minimize app to the system tray or to the window panel on App close ("**Hide app window to tray on App close**") added in release https://github.com/clesssalvein/GlpiClient/releases/tag/v0.305.

When the option is **unchecked** - the app window will be minimized to the window panel if you close it. And if you try to run another instance of the app - app window will be maximized.
In case if this option is **checked** - there still will be appear a message "The app is already running".
Please test it.
